vars/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 # vars file for openvpn
 openvpn_config_file: "openvpn_{{ openvpn_proto }}_{{ openvpn_port }}"
+openvpn_use_crl: true
```

Comment:
3 things: 
1. Put this in `defaults/main.yml` - this is a role default as per https://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable.

2. Set it to false by default, don't want to break existing certs if people rerun this playbook.

3. Document it in https://github.com/kyl191/ansible-role-openvpn/blob/master/README.md